docs/policy/external-oasis-repos.md

```diff
@@ -31,8 +31,11 @@ OSG Responsibilities
 * OSG will provide a best-effort mirror of the full contents of the external repo.  We will attempt to provide best-effort integrity of the
   object contents, but assume users of the Stratum-1 will do further integrity checking.  No SLA is provided covering potential data corruptions.
 * OSG will provide best-effort notification to the mirrored repository in case OSG detects a service outage of the external repo.
-* In the event of a reported security incident, OSG will replace the repository contents with an empty directory, signed by an OSG-managed key.
-  This will be done under the direction of the security team.
+* In the event of a security incident, the operations group at Indiana University
```

Comment:
Why limit this to Indiana?

Reply:
Agreed. Based on our discussion about Operations teams at the November 1st Area Coordinators meeting, it seems like this point should be defined in terms of a role, a responsibility, or a capability. Think SciTokens!